huggingface/azureml/hf-ort.py

```diff
@@ -0,0 +1,146 @@
+import os
+import requests
+import sys
+import shlex
+from datetime import datetime
+import argparse
+
+# AzureML libraries
+import azureml.core
+from azureml.core import Experiment, Workspace, Datastore, Run, Environment
+from azureml.core.compute import ComputeTarget, AmlCompute, AksCompute
+from azureml.core.compute_target import ComputeTargetException
+from azureml.core import ScriptRunConfig
+from azureml.core.runconfig import PyTorchConfiguration
+
+# Check core SDK version number
+print("SDK version:", azureml.core.VERSION)
+
+print("The arguments are: " + str(sys.argv))
+
+parser = argparse.ArgumentParser()
+
+parser.add_argument("--workspace_name",
+                        help="Name of the AzureML workspace", type=str, required=True)
+
+parser.add_argument("--resource_group",
+                        help="Resource group of the AzureML workspace", type=str, required=True)
+
+parser.add_argument("--subscription_id",
+                        help="Subscription of the AzureML workspace", type=str, required=True)
+
+parser.add_argument("--gpu_cluster_name",
+                        help="GPU cluster for the script to run on", type=str, required=True)
+
+parser.add_argument("--min_cluster_node",
+                        help="Min number of node for new cluster", type=str, required=True)
+
+parser.add_argument("--max_cluster_node",
+                        help="Max number of node for new cluster", type=str, required=True)
+
+parser.add_argument("--hf_model",
+                        help="Huggingface models to run. One of 'bert-large', 'distilbert-base', 'gpt2', 'bart-large', 't5-large'", type=str, required=True)
+
+parser.add_argument("--run_config",
+                        help="One of pt-fp16, ort, ds_s0, ds_s0_ort, ds_s1, ds_s1_ort", type=str, required=True)
+
+parser.add_argument("--model_batchsize",
+                        help="Model batchsize", type=str, required=False)
+
+parser.add_argument("--process_count",
+                        help="GPU process count", type=str, required=False, default=8)
+
+parser.add_argument("--node_count",
+                        help="node count", type=str, required=False, default=1)
+
+args = parser.parse_args()                  
+
+ws = Workspace.get(name=args.workspace_name, subscription_id=args.subscription_id, resource_group=args.resource_group)
+
+# Create the compute cluster
+gpu_cluster_name = args.gpu_cluster_name
+
+# Verify that the cluster doesn't exist already
+try:
```

Comment:
Suggestion: What about using

```
target = ws.compute_targets[args.gpu_cluster_name]
```

and throw an error if it doesn't exist. Your example isn't designed to teach them about creating compute clusters.

Reply:
That's a good point. Removed the part to create compute now